src/panels/lovelace/editor/config-elements/hui-view-editor.ts

```diff
@@ -28,7 +28,8 @@ export class HuiViewEditor extends hassLocalizeLitMixin(LitElement) {
     if (!this._config) {
       return "";
     }
-    return this._config.id || "";
+
+    return "id" in this._config ? this._config.id! : "";
```

Comment:
This will still be empty with an id of 0, should be:
```suggestion
    return "id" in this._config ? String(this._config.id!) : "";
```

Reply:
I checked it
Why would it be empty? ITs not an or statement. No reason to convert to string

Reply:
I see in the viewID this is needed but the editor will take the number of 0

Reply:
Yeah input will convert anything you put in it to string...